marketplace/deployer_helm_tiller_base/bin/deploy_with_tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
It only makes sense to have these files in the deployer image if it's also used for running the hooks. I think that's a good idea. See my comment below about supporting a property for passing back the deployer image to the solution.

Reply:
see other comment for discussion.

Reply:
(Replying here as I can't reply on the other thread--seems like the commit was gone)

Note that we have a requirement that all images have to come from `marketplace.gcr.io`. Pointing this to a base deployer requires us publishing that to `marketplace.gcr.io` separately. I think it's more work for no benefits. Since the version of the deployer image is guaranteed to be consistent with that of the application helm chart, it seems more straight-forward and fool-proof to use that image.

The caveat is that it requires some boiler-plate in `schema.yaml` to plumb the deployer image through.

Reply:
(Oops - pull rebase'd from `master`. Maybe I ought not to do that. Here's that comment for posterity:

> i don't think it's necessary to provide a default for `image` property.
> 
> in a follow up pull request, i'd like to set it to the base helm tiller deployer image, but that will require a little more leg work on our release process to associate chart releases with deployer image releases.)

A couple things aren't clear to me - let's sync offline.

Reply:
Resolved offline. For the record, we want to use the (application) deployer image as the container to run the hooks.